app/models/conversation.rb

```diff
@@ -116,10 +116,6 @@ def summary
     analysis[:summary]
   end
 
-  def tags
```

Comment:
I think this was maybe missed in conflict resolution, but [there's a duplicate method](https://github.com/magma-labs/magma-chat/commit/79cec9417a88c6914c5938866c758cba39aa5f38) that handles an edge case better.